vmdb/app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -580,7 +582,9 @@ def schedule_set_record_vars(schedule)
       }
     end
 
-    schedule.run_at ||= Hash.new
+    return if params[:action] == "db_backup"
```

Comment:
Same question here.  I don't want to be checking action names in the controller.